### ISSUE TITLE
the color of the grids do not follow the focus move

### ISSUE THREAD

**Issue by pankaj-kumar34** [carbon-design-system/carbon#6143]
In chrome, under Service credentials section, use the Tab and Shift+Tab keys to navigate to and back ,when the focus left ,the dialog and button lost. please refer to below .
WCAG 2.1:Use the Tab and Shift+Tab keys to navigate to and back between widgets
Checkpoint 4.1.2 reflow：https://www.ibm.com/able/guidelines/ci162/name_role_value.html
![image](https://user-images.githubusercontent.com/18532515/82975453-1ae3d280-9ffa-11ea-833d-9d106da0ea17.png)

**Comment by emyarod** [comment]
can you create a [reduced test case in Code Sandbox](https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox) for easier testing and debugging?

**Comment by pankaj-kumar34** [comment]
https://codesandbox.io/s/awesome-bassi-n49hv?file=/src/index.js

Focus issue, we are able to make it work but when we switch tab using arrow button then the values are not coming as expected.

**Comment by dakahn** [comment]
Can you recreate the issue in that sandbox you posted or is this just on the site you're testing? I'm unable to recreate this bug. Focus follows selected switcher button

**Comment by pankaj-kumar34** [comment]
In image below last printed value is index=2, which is value for _Month_ switch button. But Focused button is _Day_.
![image](https://user-images.githubusercontent.com/18532515/95043510-91a48100-06fa-11eb-961e-7c2e3c4cbc0c.png)

**Comment by tw15egan** [comment]
related: https://github.com/carbon-design-system/carbon/issues/6162

PR: https://github.com/carbon-design-system/carbon/pull/6968

**Comment by joshblack** [comment]
Closed by: https://github.com/carbon-design-system/carbon/pull/6257
